src/extension.ts

```diff
@@ -130,6 +130,7 @@ export async function activate(context): Promise<extensionapi.ExtensionAPI> {
             maybeRunKubernetesCommandForActiveWindow.bind(this, 'create', "Kubernetes Creating...")
         ),
         registerCommand('extension.vsKubernetesDelete', deleteKubernetes),
+        registerCommand('extension.vsKubernetesDeleteNow', deleteKubernetesNow),
```

Comment:
Our naming convention for these functions is a bit weird (and we should probably make it more natural) - we typically use the command name followed by `Kubernetes` - so I'd suggest `deleteNowKubernetes`.